recipes/qtxlsxwriter/all/CMakeLists.txt

```
@@ -0,0 +1,87 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(qtxlsxwriter)
+
+find_package(qt REQUIRED CONFIG)
+
+set(QTXLSXWRITER_TARGET_NAME qtxlsxwriter)
+
+include(conan_qt_executables_variables)
+set(CMAKE_AUTOMOC ON)
+
+file(GLOB QTXLSXWRITER_HEADERS LIST_DIRECTORIES=false source_subfolder/src/xlsx/*.h)
+file(GLOB QTXLSXWRITER_SOURCES LIST_DIRECTORIES=false source_subfolder/src/xlsx/*.cpp)
```

You're glob does not pick up directories

```suggestion
file(GLOB QTXLSXWRITER_HEADERS source_subfolder/src/xlsx/*.h)
file(GLOB QTXLSXWRITER_SOURCES source_subfolder/src/xlsx/*.cpp)
```

---

Since by default [`GLOB` lists directories](https://cmake.org/cmake/help/v3.20/command/file.html#glob), I prefer to set this option to false to always exclude them.